src/shared/HeroSection/HeroSectionView.tsx

```diff
@@ -7,15 +8,20 @@ interface Props {
   children: React.ReactNode;
 }
 
-const HeroView: React.FC<Props> = ({ children }) => (
-  <Container>
-    <BackgroundImage
-      src={backgroundImage.src}
-      srcSet={backgroundImage.srcSet}
-    />
-    <Overlay as="div" />
-    <Inner>{children}</Inner>
-  </Container>
-);
+const HeroView: React.FC<Props> = ({ children }) => {
+  const { t } = useTranslation("home");
+
+  return (
+    <Container>
+      <BackgroundImage
+        src={backgroundImage.src}
+        srcSet={backgroundImage.srcSet}
+        alt={t("hero.altBackground")}
```

Comment:
For decorative images, alt tag value should [be left out empty](https://www.w3.org/WAI/tutorials/images/decorative/).